### PR TITLE
JAMES-2984 Conflict port 8000 binding between james and swift service

### PR DESCRIPTION
**Pull request by trantienduchn**

### CODE REVIEW

**Comment by trantienduchn** [comment]
```
[9b05037c97538e58bdfe0907376d533ccf5084f4] [ERROR] Errors: 
[9b05037c97538e58bdfe0907376d533ccf5084f4] [ERROR]   ConcurrentTestRunnerTest.runShouldPerformAllOperationsEvenOnOccasionalExceptions:230 » NotTerminated
```
test this please

File: dockerfiles/run/docker-compose.yml [hunk 1]
```diff
@@ -46,5 +46,4 @@ services:
     image: linagora/openstack-keystone-swift:pike
     ports:
       - "5000:5000"
-      - "8080:8080"
```
Comment (Arsnael) [inline dockerfiles/run/docker-compose.yml]: That's port `8080`, not `8000` ?

Comment (Arsnael) [reply dockerfiles/run/docker-compose.yml]: Are you sure you are not in conflict with something else unrelated instead?

**Comment by trantienduchn** [comment]
8080 != 8000

**Pull request closed without merging**